jwst/tests/base_classes.py

```diff
@@ -1,4 +1,5 @@
 from glob import glob as _sys_glob
+import logging
```

Comment:
We should probably remove logging from this module.

Reply:
Left in from debugging the issue